### PR TITLE
Fixed Space Issue

### PR DESCRIPTION
**Pull request by Maxhar30**
PR for issue [12](https://github.com/foretheta/Yassine/issues/12)

### CODE REVIEW

File: dist/index.js [hunk 1]
```diff
@@ -8471,14 +8471,14 @@ try {
   octokit.rest.issues.createMilestone({
     owner: github.context.repo.owner,
     repo: github.context.repo.repo,
-    title: "Sprint(" + currentMonth + "/" + currentYear + ")-a",
+    title: "Sprint (" + currentMonth + "/" + currentYear + ")-a",
```
Comment (Maxhar30) [inline dist/index.js:8474]: This issue was raised from this Section

File: dist/index.js [hunk 2]
```diff
@@ -8471,14 +8471,14 @@ try {
...
     due_on: MID_OF_THE_MONTH_DATE.toISOString(),
   })
 
   octokit.rest.issues.createMilestone({
     owner: github.context.repo.owner,
     repo: github.context.repo.repo,
-    title: "Sprint(" + currentMonth + "/" + currentYear + ")-b",
+    title: "Sprint (" + currentMonth + "/" + currentYear + ")-b",
```
Comment (Maxhar30) [inline dist/index.js:8481]: And also this one

**Review by danialranjha — APPROVED** [review]
ship it!

**Comment by vuongbui** [comment]
sorry for late response. After discussing with Mazhar about this issue, I troubleshooted the npm and found that the dist folder is a distribution version created by running `npm build` command.
This feature is in npm@v6 and not available in recent npm versions.
So I create the issue https://github.com/foretheta/milestone-actions/issues/8 to refactor the code.

**Pull request merged**